Description: Fix fails to display zip file contents after unzip update
Bug-Debian: http://bugs.debian.org/528239
Author: Johannes Stezenbach <js@sig21.net>
Author: Yury V. Zaytsev <yury@shurup.com>

--- mc-4.7.0.3.orig/lib/vfs/mc-vfs/extfs/uzip.in	2010-02-25 11:51:39.000000000 +0100
+++ mc-4.7.0.3/lib/vfs/mc-vfs/extfs/uzip.in	2010-03-17 10:43:37.000000000 +0100
@@ -19,7 +19,7 @@
 # Location of the unzip program
 my $app_unzip = "@UNZIP@";
 # Set this to 1 if zipinfo (unzip -Z) is to be used (recommended), otherwise 0.
-my $op_has_zipinfo = @HAVE_ZIPINFO@;
+my $op_has_zipinfo = 1;
 
 # Command used to list archives (zipinfo mode)
 my $cmd_list_zi = "$app_unzip -Z -l -T";
